pages/en/lb2/readmes/loopback-example-connector.md

```diff
@@ -0,0 +1,33 @@
+# loopback-example-connector.soap
```

Comment:
This file is spurious and duplicates loopback-example-connector-soap.md.  Not sure how it got generated, but it should be removed.